dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/GroupResource.java

```diff
@@ -250,12 +259,12 @@ public Response export(@ApiParam(hidden = true)
                            @QueryParam("_type") String resourceTypes,
                            @ApiParam(value = "Output format of requested data", allowableValues = FHIR_NDJSON, defaultValue = FHIR_NDJSON)
                            @QueryParam("_outputFormat") String outputFormat,
-                           @ApiParam(value = "Request data that has been updated after the given point. (Not implemented yet)", hidden = true)
+                           @ApiParam(value = "Resources updated after this period will be included in the response. Must be a FHIR date-time.")
```

Comment:
```suggestion
                           @ApiParam(value = "Resources updated after this time will be included in the response. Must be a FHIR date-time.")
```
This should probably be something that implies `instant` rather than a date range.

Reply:
This text is actually from a version of the FHIR spec. I'm replacing it with the current version which is more precise. Also, the real change I needed to do was move from String to a InstantDt